sycl/test/check_device_code/native_cpu/kernelhandler-scalar.cpp

```diff
@@ -1,3 +1,4 @@
+// REQUIRES: native_cpu
```

Comment:
Could you change the Native CPU tests to use `-fno-sycl-libspirv -Wno-unsafe-libspirv-not-linked` like you did the `math-builtins` test, rather than adding `REQUIRES`?

Reply:
Thanks. My thinking there was we don't enable Native CPU in most of CI yet, so by adding the `REQUIRES`, we would reduce our testing. With this, the update of the tests looks fine to me.